src/rules/__tests__/max-nested-describe.test.ts

```diff
@@ -0,0 +1,135 @@
+import { TSESLint } from '@typescript-eslint/experimental-utils';
+import dedent from 'dedent';
+import resolveFrom from 'resolve-from';
+import rule from '../max-nested-describe';
+
+const ruleTester = new TSESLint.RuleTester({
+  parser: resolveFrom(require.resolve('eslint'), 'espree'),
+  parserOptions: {
+    ecmaVersion: 2017,
+  },
+});
+
+ruleTester.run('max-nested-describe', rule, {
```

Comment:
Could you add a test for `{ max: 0 }`

Reply:
Could you chuck in a few tests for prefixed & modified describes? (i.e `fdescribe`, `describe.skip` - feel free to just chuck these into some of the existing tests), and one of each version of `describe.each`.